src/ducks/user/sagas.ts

```diff
@@ -11,22 +11,25 @@ import {
 
 import { fetchUser, fetchUserFailed, fetchUserSuccess } from './index';
 import { FetchUserPayload } from './interfaces';
-import { get } from './services';
+import { get, reload } from './services';
 
 export function* fetchUserSaga(): Generator<
+  | CallEffect<void>
   | CallEffect<User>
   | PutEffect<PayloadAction<FetchUserPayload>>
   | PutEffect<PayloadAction<string>>,
   void,
   User
 > {
   try {
+    yield call(reload);
     const response = yield call(get);
     yield put(
       fetchUserSuccess({
         username: response.displayName ?? '',
         email: response.email ?? '',
         avatar: response.photoURL ?? '',
+        isEmailVerified: response.emailVerified ?? '',
```

Comment:
emailVerified has boolean type, but here you write empty string, to fix this, you just need to do interfaces for initial state

Reply:
Anyway it casts to boolean and interface is already there, but i change fallback value to false to more logic behavior